lib/sphero.js

```diff
@@ -24,6 +24,19 @@ function classCallCheck(instance, Constructor) {
   }
 }
 
+/**
+ * Creates a new sphero instance
+ *
+ * @constructor
+ * @param {String} address of the connected sphero
+ * @param {Object} opts for sphero setup
+ *   @param {Object} [opts.adaptor=serial] sets the adaptor for the connection
+ *   @param {Object} [opts.sop2=0xFD] sop2 to be passed to commands
+ *   @param {Object} [opts.timeout=500] deadtime between commands, in ms
+ * @example
```

Comment:
These don't need to be indented, and `opts.adaptor` doesn't really have a default, guess it would be better specified in the description that it defaults to serial port?